apps/web-giddh/src/assets/css/style-2.scss

```diff
@@ -9454,10 +9455,10 @@ ngx-daterangepicker-material {
 #HW_badge_cont {
     visibility: visible !important;
     width: auto !important;
-    top: -5px;
+    top: 1px;
```

Comment:
Why change this it part of header ?

Reply:
Yes, this is part of the header, but the button is not working when clicked below. Please check it on the test.